src/ui/screen/screen_mod_manager.cpp

```diff
@@ -0,0 +1,351 @@
+         event.type == EVENT_INTERFACE_BACK    ||
+         (event.type == EVENT_KEY_DOWN && event.GetData<KeyEventData>()->key == KEY(ESCAPE)) )
+    {
+        m_main->ChangePhase(PHASE_MAIN_MENU);
+        std::string title, htext, text;
+        GetResource(RES_TEXT, RT_TITLE_APPNAME, title);
+        GetResource(RES_TEXT, RT_DIALOG_MODSCHANGE_TITLE, htext);
+        GetResource(RES_TEXT, RT_DIALOG_MODSCHANGE_TEXT, text);
+        m_dialog->StartInformation(title, htext, text);
+        return false;
+    }
+
+    if (event.type == EVENT_INTERFACE_LOAD)
+    {
+        pl = static_cast<CList*>(pw->SearchControl(EVENT_INTERFACE_MODLIST_UNLOADED));
+        if ( pl == nullptr )  return false;
+        modName = pl->GetItemName(pl->GetSelect());
+        modPath = modPathRaw.c_str();
+        boost::filesystem::rename(modPath+OFF+modName, modPath+modName);
+        m_main->ChangePhase(PHASE_MOD_MANAGER);
+    }
+
+    if (event.type == EVENT_INTERFACE_UNLOAD)
+    {
+        pl = static_cast<CList*>(pw->SearchControl(EVENT_INTERFACE_MODLIST_LOADED));
+        if ( pl == nullptr )  return false;
+        modName = pl->GetItemName(pl->GetSelect());
+        modPath = modPathRaw.c_str();
+        boost::filesystem::rename(modPath+modName, modPath+OFF+modName);
+        m_main->ChangePhase(PHASE_MOD_MANAGER);
+    }
+
+    switch (event.type)
+    {
+        case EVENT_INTERFACE_MODLIST_LOCAL:
+            m_global = false;
+            m_main->ChangePhase(PHASE_MOD_MANAGER);
+            break;
+
+        case EVENT_INTERFACE_MODLIST_GLOBAL:
+            m_global = true;
+            m_main->ChangePhase(PHASE_MOD_MANAGER);
+            break;
+
+        case EVENT_INTERFACE_MODLIST_UNLOADED:
+            pl = static_cast<CList*>(pw->SearchControl(EVENT_INTERFACE_MODLIST_LOADED));
+            if ( pl == nullptr )  break;
+            pb = static_cast<CButton*>(pw->SearchControl(EVENT_INTERFACE_UNLOAD));
+            if ( pb == nullptr )  break;
+            pl->SetSelect(-1);
+            pb->ClearState(STATE_ENABLE);
+            pb = static_cast<CButton*>(pw->SearchControl(EVENT_INTERFACE_LOAD));
+            if ( pb == nullptr )  break;
+            pb->SetState(STATE_ENABLE);
+            break;
+
+        case EVENT_INTERFACE_MODLIST_LOADED:
+            pl = static_cast<CList*>(pw->SearchControl(EVENT_INTERFACE_MODLIST_UNLOADED));
+            if ( pl == nullptr )  break;
+            pb = static_cast<CButton*>(pw->SearchControl(EVENT_INTERFACE_LOAD));
+            if ( pb == nullptr )  break;
+            pl->SetSelect(-1);
+            pb->ClearState(STATE_ENABLE);
+            pb = static_cast<CButton*>(pw->SearchControl(EVENT_INTERFACE_UNLOAD));
+            if ( pb == nullptr )  break;
+            pb->SetState(STATE_ENABLE);
+            break;
+
+        case EVENT_INTERFACE_MODS_DIR:
+            modPathRaw = CResourceManager::GetSaveLocation() + "/" + "mods";
+            #ifdef _WIN32
```

Comment:
Don't use these defines, use [our own PLATFORM_* defines](https://github.com/colobot/colobot/blob/master/src/common/config.h.cmake#L4-L8) instead. Or, even better, proxy these calls through CSystemUtils.

Reply:
Done in 3221b81 .